sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -14,43 +14,50 @@
     </item>
     <item>
       <EFDB_Name>timestamp</EFDB_Name>
-      <Description>Time at which the data was determined (TAI unix seconds).</Description>
+      <Description>Time at which the most recent data was read (TAI unix seconds).</Description>
       <IDL_Type>double</IDL_Type>
       <Units>second</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>ux</EFDB_Name>
-      <Description>x-axis wind speed.</Description>
+      <EFDB_Name>speed</EFDB_Name>
+      <Description>Median wind speed (x, y, z) measured over a period on the order of a minute.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>m/s</Units>
-      <Count>1</Count>
+      <Count>3</Count>
+    </item>
+    <item>
+      <EFDB_Name>speedStdDev</EFDB_Name>
+      <Description>Standard deviation of wind speed (x, y, z) measured over a period on the order of a minute.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s</Units>
+      <Count>3</Count>
     </item>
     <item>
-      <EFDB_Name>uy</EFDB_Name>
-      <Description>y-axis wind speed.</Description>
+      <EFDB_Name>speedMagnitude</EFDB_Name>
+      <Description>Median wind speed magnitude measured over a period on the order of a minute.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>m/s</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>uz</EFDB_Name>
-      <Description>z-axis wind speed.</Description>
+      <EFDB_Name>speedMaxMagnitude</EFDB_Name>
+      <Description>Maximum wind speed magnitude measured over a period on the order of a minute.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>m/s</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>ts</EFDB_Name>
-      <Description>Sonic temperature (air temperature measured sonically). This has poor absolute accuracy (it can be off by several degrees) but good time resolution.</Description>
+      <EFDB_Name>sonicTemperature</EFDB_Name>
+      <Description>Sonic temperature (air temperature measured sonically); median of values measured over a period on the order of a minute. This has poor absolute accuracy (it can be off by several degrees) but good time resolution.</Description>
       <IDL_Type>float</IDL_Type>
       <Units>deg_C</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>status</EFDB_Name>
-      <Description>Device status. The details depend on the device, but 0 always means that no problems have been detected.</Description>
-      <IDL_Type>short</IDL_Type>
+      <EFDB_Name>isok</EFDB_Name>
+      <Description>Is the data valid?</Description>
```

Comment:
How is this different than the "sensorStatus" event? It feels redundant to have this "isok" item here and for all other sensors where this has been added.